app/_src/guides/consumer-producer-policies.md

```diff
@@ -0,0 +1,262 @@
+---
+title: Namespace scoped policies
+---
+
+## Prerequisites
+
+- Completed [quickstart](/docs/{{ page.version }}/quickstart/kubernetes-demo/) to set up a zone control plane with demo application
+
+## Basic setup
+
+In order to be able to fully utilize namespace scoped policies you need to use real [MeshService](/docs/{{ page.version }}/networking/meshservice). 
+To enable MeshService resource generation on Mesh level we need to run:
+
+```shell
+echo "apiVersion: kuma.io/v1alpha1
+kind: Mesh
+metadata:
+  name: default
+spec:
+  meshServices:
+    enabled: Everywhere
+  mtls:
+    enabledBackend: ca-1
+    backends:
+    - name: ca-1
+      type: builtin" | kubectl apply -f -
+```
+
+We also need HostnameGenerator for easy access to our services:
+
+```shell
+echo 'apiVersion: kuma.io/v1alpha1
+kind: HostnameGenerator
+metadata:
+  name: all
+  namespace: {{site.mesh_namespace}}
+  labels:
+    kuma.io/mesh: default
+spec:
+  selector:
+    meshService:
+      matchLabels:
+        k8s.kuma.io/namespace: kuma-demo
+  template: "{{ .DisplayName }}.{{ .Namespace }}.mesh"' | kubectl apply -f -
+```
+
+With this HostnameGenerator we will be able to call demo-app using dns name `demo-app.kuma-demo.mesh`.
+To make sure that traffic works in our examples let's configure MeshTrafficPermission to allow all traffic:
+
+```shell
+echo "apiVersion: kuma.io/v1alpha1
+kind: MeshTrafficPermission
+metadata:
+  namespace: {{site.mesh_namespace}}
+  name: mtp
+spec:
+  targetRef:
+    kind: Mesh
+  from:
+    - targetRef:
+        kind: Mesh
+      default:
+        action: Allow" | kubectl apply -f -
+```
+
+To finish the setup we need to create two additional namespaces with sidecar injection for clients we will be using to communicate 
+with our demo-app:
+
+```shell
+echo "apiVersion: v1
+kind: Namespace
+metadata:
+  name: first-consumer
+  labels:
+    kuma.io/sidecar-injection: enabled
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: second-consumer
+  labels:
+    kuma.io/sidecar-injection: enabled" | kubectl apply -f -
+```
+
+Now we can create deployment we will be using to communicate with our demo-app from a first-consumer namespace:
+
+```shell
+kubectl run consumer --image nicolaka/netshoot -n first-consumer --command -- /bin/bash -c "while true; do ping localhost; sleep 60;done"
+```
+
+and from the second-consumer namespace:
+
+```shell
+kubectl run consumer --image nicolaka/netshoot -n second-consumer --command -- /bin/bash -c "while true; do ping localhost; sleep 60;done"
+```
+
+You can make now make a couple of requests to our demo app to check if everything is working: 
+
+```shell
+kubectl exec -n first-consumer consumer -- curl --no-progress-meter demo-app.kuma-demo.mesh:5000/counter
```

Comment:
I'd use `kubectl exec -n first-consumer consumer -- curl -s -XPOST demo-app.kuma-demo:5000/increment` 

A few things different here:
- use the k8s hostname because that enables the guide to introduce less information
- use `-s` instead of `--no-progress-meter` it's quieter
- use `-X POST /increment` I find it more straightforward when calls do something.

Reply:
makes sense, changed as requested